README.md

```diff
@@ -52,6 +52,16 @@ access to some shared credentials and accounts.
    > script/server
    ```
 
+1. Setup the database:
+
+   ```shell
+   > script/db-create
+   ```
+
+   ```shell
```

Comment:
Migrations will run when you start the server through `script/server`. I think we can remove this.

Reply:
Interesting, I see that now. I was only starting the DB since I didn't want to bother with building the web container to just work on the data loading stuff. I wonder if it should be another section then?

Reply:
Maybe we can add this as an optional step with a note that starting the server will automatically migrate the db.